cmp/project.py

```diff
@@ -709,53 +718,61 @@ def init_eeg_project(project_info, bids_layout, is_new_project, gui=False, debug
             eeg_pipeline.input_folders,
             session=project_info.subject_session
         )
+        subject_derivatives_directory = os.path.join(
+            derivatives_directory, project_info.subject, project_info.subject_session
+        )
     else:
         if debug:  # pragma: no cover
             print('Refresh folder WITHOUT session')
-        refresh_folder(bids_directory, derivatives_directory,
-                       project_info.subject, eeg_pipeline.input_folders)
-
-    eeg_inputs_checked = eeg_pipeline.check_input(layout=bids_layout, gui=gui)
-    if eeg_inputs_checked:
-        if is_new_project and eeg_pipeline is not None:
-            print("> Initialize eeg project")
-            if not os.path.exists(derivatives_directory):
-                try:
-                    os.makedirs(derivatives_directory)
-                except os.error:  # pragma: no cover
-                    print("... Info: %s was already existing" %
-                          derivatives_directory)
-                finally:
-                    print("... Info : Created directory %s" %
-                          derivatives_directory)
-
-            if (project_info.subject_session != '') and (project_info.subject_session is not None):
-                project_info.eeg_config_file = os.path.join(
-                    derivatives_directory, '%s_%s_eeg_config.ini' % (project_info.subject,
-                                                                     project_info.subject_session))
-            else:
-                project_info.eeg_config_file = os.path.join(
-                    derivatives_directory, '%s_eeg_config.ini' % project_info.subject)
+        refresh_folder(
+            bids_directory,
+            derivatives_directory,
+            project_info.subject,
+            eeg_pipeline.input_folders
+        )
+        subject_derivatives_directory = os.path.join(
+            derivatives_directory, project_info.subject
+        )
 
-            if os.path.exists(project_info.eeg_config_file):
-                warn_res = project_info.configure_traits(view='eeg_warning_view')
-                if warn_res:
-                    eeg_save_config(eeg_pipeline, project_info.eeg_config_file)
-                else:
-                    return None
-            else:
-                eeg_save_config(eeg_pipeline, project_info.eeg_config_file)
+    if is_new_project and eeg_pipeline is not None:
+        print("> Initialize eeg project")
+        if not os.path.exists(subject_derivatives_directory):
+            try:
+                os.makedirs(subject_derivatives_directory)
+            except os.error:  # pragma: no cover
+                print("... Info: %s was already existing" %
+                      subject_derivatives_directory)
+            finally:
+                print("... Info : Created directory %s" %
+                      subject_derivatives_directory)
```

Comment:
Same here, print statement should go into the try block

Reply:
Addressed by commit https://github.com/connectomicslab/connectomemapper3/pull/149/commits/6c39c8e1cf3969044da41122831f9a7cf03974b4